### PR TITLE
Apply strip_unused on inference graph

### PR DESCRIPTION
**Pull request by lissyx**

### CODE REVIEW

**Comment by lissyx** [comment]
This was recommended on the issue filed about TensorFlow CoreML exporter, but I can't see any difference for the moment.

**Comment by reuben** [comment]
Does this change our graph at all? E.g does it change its checksum?

**Comment by lissyx** [comment]
> Does this change our graph at all? E.g does it change its checksum?

Unfortuntaly, the list of ops does not change. So I'm unsure if we should do that. On the other hand, it's likely a good idea to apply it always.

**Comment by lissyx** [comment]
```
$ sha1sum ~/tmp/deepspeech/0.5.0-ckpt/output_graph.*
ebb290d1cfb1f7bae8268a81dead34d9d1ca5d3e  /home/alex/tmp/deepspeech/0.5.0-ckpt/output_graph.pb
5222826d794d060988c17ac80fdc1d18272377f2  /home/alex/tmp/deepspeech/0.5.0-ckpt/output_graph.stripped.pb
```

So the checksum does change.

**Comment by lissyx** [comment]
> ```
> $ sha1sum ~/tmp/deepspeech/0.5.0-ckpt/output_graph.*
> ebb290d1cfb1f7bae8268a81dead34d9d1ca5d3e  /home/alex/tmp/deepspeech/0.5.0-ckpt/output_graph.pb
> 5222826d794d060988c17ac80fdc1d18272377f2  /home/alex/tmp/deepspeech/0.5.0-ckpt/output_graph.stripped.pb
> ```
> 
> So the checksum does change.

Well, two consecutive runs of the same export do produce different sha1, so it's not really conclusive.

File: DeepSpeech.py [hunk 1]
```diff
@@ -723,6 +725,13 @@ def do_graph_freeze(output_file=None, output_node_names=None, variables_blacklis
                 variable_names_blacklist=variables_blacklist,
                 initializer_nodes='')
 
+            input_node_names = []
+            return strip_unused_lib.strip_unused(
+                input_graph_def=frozen,
+                input_node_names=input_node_names,
+                output_node_names=output_node_names.split(','),
+                placeholder_type_enum=dtypes.float32.as_datatype_enum)
```
Comment (reuben) [inline DeepSpeech.py]: You shouldn't need to go through `tensorflow.dtypes.float32`, just `tf.float32` should be enough.

File: DeepSpeech.py [hunk 2]
```diff
@@ -19,6 +19,8 @@
 from evaluate import evaluate
 from six.moves import zip, range
 from tensorflow.python.tools import freeze_graph
+from tensorflow.python.tools import strip_unused_lib
```
Comment (reuben) [inline DeepSpeech.py]: Merge with import above?

**Pull request merged**